src/app/features/approver/approver.stories.tsx

```diff
@@ -0,0 +1,116 @@
+    </Flex>
+  ),
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Approver>;
+
+export const ExampleOne: Story = {
+  args: {
+    children: (
+      <>
+        <Approver.Header title="Some prompt that breaks two lines" requester="gamma.io" />
+        <Callout title="Some callout">Hey watch out for this sketchy app</Callout>
+        <Approver.Section>
+          <Approver.Subheader>Example flag content</Approver.Subheader>
+          <Flag img={<Circle size="40px" backgroundColor="ink.border-default" />} align="top">
+            <Box width="90%" height="16px" backgroundColor="ink.border-default" />
+            <Box width="75%" height="12px" backgroundColor="ink.border-default" mt="space.02" />
+          </Flag>
+        </Approver.Section>
+        <Approver.Section>
+          <Approver.Subheader>Example rich content with avatar</Approver.Subheader>
+          <ItemLayout
+            titleLeft="Michelly token"
+            titleRight="100 MICA"
+            captionLeft="SIP-10"
+            captionRight="$894,891"
+            flagImg={<Circle size="40px" backgroundColor="ink.border-default" />}
+          />
+        </Approver.Section>
+        <Approver.Advanced>
+          <Approver.Section>
+            <Approver.Subheader>In the advanced section</Approver.Subheader>
+            <ItemInteractive onClick={() => {}} mt="space.03" mb="space.03">
+              <ItemLayout
+                titleLeft="Pressable"
+                titleRight="Mr. Clicky"
+                captionLeft="Interactive item"
+                captionRight="Click me"
+                flagImg={<Circle size="40px" backgroundColor="ink.border-default" />}
+              />
+            </ItemInteractive>
+          </Approver.Section>
+          <Approver.Section>
+            <Approver.Subheader>Inputs & Outputs</Approver.Subheader>
+            <Flag align="top" img={<Circle size="40px" backgroundColor="ink.border-default" />}>
+              <Box width="100%" height="20px" backgroundColor="ink.border-default" />
+            </Flag>
+          </Approver.Section>
+          <Approver.Section>
+            <Approver.Subheader>Transaction raw data</Approver.Subheader>
+            <Flag align="top" img={<Circle size="40px" backgroundColor="ink.border-default" />}>
+              <Box width="100%" height="20px" backgroundColor="ink.border-default" />
+            </Flag>
+          </Approver.Section>
+          <Approver.Section>
+            <Approver.Subheader>Additional info</Approver.Subheader>
+            <Flag align="top" img={<Circle size="40px" backgroundColor="ink.border-default" />}>
+              <Box width="100%" height="20px" backgroundColor="ink.border-default" />
+            </Flag>
+          </Approver.Section>
+        </Approver.Advanced>
+
+        <Approver.Actions
+          actions={
+            <>
+              <Button variant="outline">Cancel</Button>
+              <Button>Approve</Button>
```

Comment:
just thinking in most cases there will be these `cancel/approve` btns no? maybe create them as default in Approver.Actions? 
but this means we need to pass approve/cancel actions as props, so quite arguable

Reply:
This is a tough one. I separated the actions (as prop) from the children, so we apply the button styles implicitly (flex with stretch) but adding the buttons themselves I'm not sure, maybe in some cases we want different ones?

Not sure how to handle exactly. Esp. if we have the animated confirm btn.

Reply:
In the case that we have behaviour such as in latest designs, e.g. 

<img width="529" alt="image" src="https://github.com/leather-wallet/extension/assets/1618764/7ceee10f-d503-49f9-af21-9feaad731f2f">

think it would need to be part of approver to share the state